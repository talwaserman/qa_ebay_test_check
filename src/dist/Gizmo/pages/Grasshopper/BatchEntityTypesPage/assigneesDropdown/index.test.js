"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _mock = require("../mock");

var _index = _interopRequireDefault(require("./index"));

it('test for generating AssigneesDropDown for update assignees when no assingee assigned to ET ', function () {
  var tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_index.default, {
    loggedUserData: _mock.mockAgent,
    currentAssigneeData: _mock.assignRowResponseMock,
    selectAssigneePlaceHolder: 'Select Assignee',
    assingeesList: _mock.getUsersResponseMock.users,
    onAssignedChanged: function onAssignedChanged() {},
    rowId: '5e94b3e79c95bfcfc44971cf'
  })).toJSON();

  expect(tree).toMatchSnapshot();
});
it('test for generating AssigneesDropDown for update assignees when assingee already assigned to ET ', function () {
  var tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_index.default, {
    loggedUserData: _mock.mockAgent,
    currentAssigneeData: _mock.assignRowResponseMock,
    selectAssigneePlaceHolder: 'Select Assignee',
    assingeesList: _mock.getUsersResponseMock.users,
    onAssignedChanged: function onAssignedChanged() {},
    rowId: '5e94b3e79c95bfcfc44971cf'
  })).toJSON();

  expect(tree).toMatchSnapshot();
});