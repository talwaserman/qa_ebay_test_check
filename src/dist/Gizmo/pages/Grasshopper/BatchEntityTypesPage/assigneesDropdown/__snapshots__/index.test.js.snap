// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for generating AssigneesDropDown for update assignees when assingee already assigned to ET  1`] = `
<div
  className="assignees-wrapper"
>
  <div
    className="assignees-dropdown"
  >
    <div
      className="ant-select-show-arrow ant-select ant-select-enabled"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      onMouseOut={[Function]}
      onMouseUp={[Function]}
      onTouchStart={[Function]}
    >
      <div
        aria-autocomplete="list"
        aria-controls="test-uuid"
        aria-expanded={false}
        aria-haspopup="true"
        className="ant-select-selection
            ant-select-selection--single"
        onKeyDown={[Function]}
        role="combobox"
        tabIndex={0}
      >
        <div
          className="ant-select-selection__rendered"
        >
          <div
            className="ant-select-selection__placeholder"
            onClick={[Function]}
            onMouseDown={[Function]}
            style={
              Object {
                "WebkitUserSelect": "none",
                "display": "none",
                "userSelect": "none",
              }
            }
            unselectable="on"
          >
            Select Assignee
          </div>
          <div
            className="ant-select-selection-selected-value"
            style={
              Object {
                "display": "block",
                "opacity": 1,
              }
            }
            title="test user"
          >
            test user
          </div>
          <div
            className="ant-select-search ant-select-search--inline"
            style={
              Object {
                "display": "none",
              }
            }
          >
            <div
              className="ant-select-search__field__wrap"
            >
              <input
                autoComplete="off"
                className="ant-select-search__field"
                onChange={[Function]}
                onKeyDown={[Function]}
                value=""
              />
              <span
                className="ant-select-search__field__mirror"
              >
                
                 
              </span>
            </div>
          </div>
        </div>
        <span
          className="ant-select-arrow"
          onClick={[Function]}
          style={
            Object {
              "WebkitUserSelect": "none",
              "userSelect": "none",
            }
          }
          unselectable="on"
        >
          <i
            aria-label="icon: down"
            className="anticon anticon-down ant-select-arrow-icon"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="down"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
              />
            </svg>
          </i>
        </span>
      </div>
    </div>
  </div>
  <div
    className="assignee-href"
  >
    <button
      className="ant-btn href-button ant-btn-link"
      onClick={[Function]}
      type="button"
    >
      <span>
        Assign to me
      </span>
    </button>
  </div>
</div>
`;

exports[`test for generating AssigneesDropDown for update assignees when no assingee assigned to ET  1`] = `
<div
  className="assignees-wrapper"
>
  <div
    className="assignees-dropdown"
  >
    <div
      className="ant-select-show-arrow ant-select ant-select-enabled"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      onMouseOut={[Function]}
      onMouseUp={[Function]}
      onTouchStart={[Function]}
    >
      <div
        aria-autocomplete="list"
        aria-controls="test-uuid"
        aria-expanded={false}
        aria-haspopup="true"
        className="ant-select-selection
            ant-select-selection--single"
        onKeyDown={[Function]}
        role="combobox"
        tabIndex={0}
      >
        <div
          className="ant-select-selection__rendered"
        >
          <div
            className="ant-select-selection__placeholder"
            onClick={[Function]}
            onMouseDown={[Function]}
            style={
              Object {
                "WebkitUserSelect": "none",
                "display": "none",
                "userSelect": "none",
              }
            }
            unselectable="on"
          >
            Select Assignee
          </div>
          <div
            className="ant-select-selection-selected-value"
            style={
              Object {
                "display": "block",
                "opacity": 1,
              }
            }
            title="test user"
          >
            test user
          </div>
          <div
            className="ant-select-search ant-select-search--inline"
            style={
              Object {
                "display": "none",
              }
            }
          >
            <div
              className="ant-select-search__field__wrap"
            >
              <input
                autoComplete="off"
                className="ant-select-search__field"
                onChange={[Function]}
                onKeyDown={[Function]}
                value=""
              />
              <span
                className="ant-select-search__field__mirror"
              >
                
                 
              </span>
            </div>
          </div>
        </div>
        <span
          className="ant-select-arrow"
          onClick={[Function]}
          style={
            Object {
              "WebkitUserSelect": "none",
              "userSelect": "none",
            }
          }
          unselectable="on"
        >
          <i
            aria-label="icon: down"
            className="anticon anticon-down ant-select-arrow-icon"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="down"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
              />
            </svg>
          </i>
        </span>
      </div>
    </div>
  </div>
  <div
    className="assignee-href"
  >
    <button
      className="ant-btn href-button ant-btn-link"
      onClick={[Function]}
      type="button"
    >
      <span>
        Assign to me
      </span>
    </button>
  </div>
</div>
`;
